src/utils/path.ts

```diff
@@ -249,6 +249,10 @@ export function join(...paths: string[]): string {
  * @returns a resolved path!
  */
 export function resolve(...paths: string[]): string {
+  /**
```

Comment:
I have no tests/additional PRs for `resolve` - so I threw a very similar comment in here for completeness sake